pkgs/native_assets_cli/lib/src/args_parser.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2024, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+String getConfigArgument(List<String> arguments) {
```

Comment:
We could actually decide that instead of making this based on arguments we set a `DART_BUILD_HOOK_CONFIG` environment variable containing the configuration.